platform/aws_test.go

```diff
@@ -1,57 +0,0 @@
-// Copyright 2015 CoreOS, Inc.
```

Comment:
In moving aws to its own amazon package, was deleting this intentional? It looks like it wasn't copied over.